frontend-html/src/tests/DateTimeCompleter.test.ts

```diff
@@ -52,7 +52,7 @@ test.each([
 });
 
 
-const dateCompleterCz = new DateCompleter(DateSequence.DayMonthYear, "DD.MM.YYYY h:mm:ss", ".",
+const dateCompleterCz = new DateCompleter(DateSequence.DayMonthYear, ".",
```

Comment:
Please add some more "CZ" tests that contain formats with separators, e.g. "9.12."